Branch-SDK/src/main/java/io/branch/referral/BranchError.java

```diff
@@ -128,6 +130,10 @@ private String initErrorCodeAndGetLocalisedMessage(int statusCode) {
         } else if (statusCode == ERR_BRANCH_TRACKING_DISABLED) {
             errorCode_ = ERR_BRANCH_TRACKING_DISABLED;
             errMsg = " Tracking is disabled. Requested operation cannot be completed when tracking is disabled";
+        } else if (statusCode == ERR_BRANCH_ALREADY_INITIALIZED) {
+            errorCode_ = ERR_BRANCH_ALREADY_INITIALIZED;
```

Comment:
A different ticket might clean this up...  errorCode almost always == statusCode.

Reply:
👍 @apeterson-branch Would you create that ticket and explain a little more?